src/main/java/picard/nio/PicardBucketUtils.java

```diff
@@ -168,34 +122,31 @@ public static boolean isGcsUrl(final IOPath pathSpec) {
     }
 
     /**
-     * @param pathSpec specifier to inspect
-     * @return true if this {@code GATKPath} represents a remote storage system which may benefit from prefetching (gcs or http(s))
+     * @param path specifier to inspect
+     * @return true if this {@code IOPath} represents a remote storage system which may benefit from prefetching (gcs or http(s))
      */
-    public static boolean isEligibleForPrefetching(final IOPath pathSpec) {
-        GATKUtils.nonNull(pathSpec);
-        return isEligibleForPrefetching(pathSpec.getScheme());
-    }
-
-    /**
-     * @param path path to inspect
-     * @return true if this {@code Path} represents a remote storage system which may benefit from prefetching (gcs or http(s))
-     */
-    public static boolean isEligibleForPrefetching(final Path path) {
+    public static boolean isEligibleForPrefetching(final IOPath path) {
         GATKUtils.nonNull(path);
-        return isEligibleForPrefetching(path.toUri().getScheme());
-    }
-
-    private static boolean isEligibleForPrefetching(final String scheme){
+        final String scheme = path.getScheme();
         return scheme != null
                 && (scheme.equals(GOOGLE_CLOUD_STORAGE_FILESYSTEM_SCHEME)
                 || scheme.equals(HTTP_FILESYSTEM_PROVIDER_SCHEME)
                 || scheme.equals(HTTPS_FILESYSTEM_PROVIDER_SCHEME));
     }
 
+    public static boolean isLocalPath(final IOPath path){
+        return path.getScheme().equals(FILE_SCHEME);
+    }
```

Comment:
This needs to happen in IOPath, because all the objects that would call the instance method isLocalPath have type IOPath, not PicardHtsPath. If you could add it to htsjdk, that would be great.